kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -177,6 +187,9 @@
         fullName: state => state.core.session.full_name,
         userId: state => state.core.session.user_id,
       }),
+      userIsLearner() {
+        return this.getUserKind == UserKinds.LEARNER;
+      },
       menuOptions() {
         return navComponents
           .filter(component => component.section === NavComponentSections.ACCOUNT)
```

Comment:
I was about to comment that we should add the filterBySoUD here as well, when I realized we could just do it at NavComponent registration:

https://github.com/learningequality/kolibri/blob/98bb185b73b247c0e774b229dd507fa6e74c4eaf/kolibri/core/assets/src/utils/navComponents.js#L39

Check for SoUD status in there ^ from plugin data, and just not add the component in the first place.

Probably good to do a logging.info when we do that so that future coders are not confused when their nav items don't show though!